OUDS/Core/Components/Sources/Link/OUDSLink.swift

```diff
@@ -122,6 +124,7 @@ public struct OUDSLink: View {
                     Image(decorative: "ic_form_chevron_left", bundle: Bundle.OUDSComponents)
                         .renderingMode(.template)
                         .resizable()
+                        .scaleEffect(layoutDirection == .rightToLeft ? -1 : 1, anchor: .center)
```

Comment:
why is a scaleEffect and not a rotateEffect because it is what it has expected ?

Reply:
Je vais tester l'autre solution ;-)

Reply:
_rotateEffect_ n'existe pas :-D